src/store/index.ts

```diff
@@ -1,10 +1,18 @@
-import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
 import { appReducer } from './app/reducer';
+import { astronautsReducer } from '../components/AstronautsPage/reducers/astronautsReducer';
+
+const reducer = combineReducers({
+  app: appReducer,
+  astronauts: astronautsReducer,
+});
 
 export const store = configureStore({
-  reducer: {
-    app: appReducer,
-  },
+  reducer,
+  middleware: getDefaultMiddleware =>
+    getDefaultMiddleware({
+      serializableCheck: false,
+    }),
```

Comment:
why do you need this middleware?

Reply:
I know what is this, but I don't know how did I get this middleware in my code.
I think it was added when I was rebasing the project.
Just removed.